subprojects/core/src/main/java/org/gradle/cache/internal/LegacyCacheCleanupEnablement.java

```diff
@@ -35,17 +35,22 @@ public LegacyCacheCleanupEnablement(GradleUserHomeDirProvider userHomeDirProvide
     public boolean isDisabledByProperty() {
         File gradleUserHomeDirectory = userHomeDirProvider.getGradleUserHomeDirectory();
         File gradleProperties = new File(gradleUserHomeDirectory, "gradle.properties");
+    
```

Comment:
You should revert this file since it's not changed actually.

Reply:
Yeah I'll do that.